content_schemas/formats/historic_appointments.jsonnet

```diff
@@ -0,0 +1,9 @@
+(import "shared/default_format.jsonnet") + {
+  definitions: {
+    details: {
+      type: "object",
+      additionalProperties: false,
+      properties: {},
```

Comment:
I'm a bit confused here - is the point of this PR just to add a basic schema, and then iterate it afterwards? Why no properties defined here?
Was imagining the shape of the schema would roughly be a list of time periods, which have titles, and then contain list of appointment details.
Same question with the other format you added

Reply:
> I'm a bit confused here - is the point of this PR just to add a basic schema, and then iterate it afterwards?

Yes, my thought was to create an empty schema then we can add to it when updating the Whitehall presenter.  I could add the fields now if that's the preferred outcome for this card?